tools/backup/backup.sh

```diff
@@ -74,6 +74,34 @@ function cloud_copy() {
     gsutil cp $backup_path $bucket_path
     gsutil cp $backup_path "${bucket_path}${LATEST_POINTER}"
     ;;
+  azure)
+    # Container is specified via BUCKET input, which can contain a path, i.e.
+    # my-container/foo
+    # AZ CLI doesn't allow this so we need to split it into container and container path.
+    CONTAINER=$(echo $BUCKET | sed 's/\/.*//')
```

Comment:
Think there is some issue in this line or the followings if you have the following configuration:
```
bucket: master-data
```
Seem to end up with an extra slash:
<img width="447" alt="Screenshot 2020-10-30 at 08 49 41" src="https://user-images.githubusercontent.com/48948282/97665908-99eba400-1a8d-11eb-8d37-2a3def33ade3.png">

<img width="444" alt="Screenshot 2020-10-30 at 08 49 17" src="https://user-images.githubusercontent.com/48948282/97665914-9d7f2b00-1a8d-11eb-95d1-c894ee6af6fd.png">

Reply:
Did a little bit more testing
```
--set bucket=master-data/testfolder/
--set bucket=master-data)/testfolder2
--set bucket=master-data/
```

And only /testfolder/ is working as expected and I think it would be reasonable to be able to have the structure in the root. 

testfolder2 ends up under the no-name folder and with the just / it gets two <no name> folders and reason can be seen in logs

Azure storage blob copy to master-data :: //system/system-2020-10-30-07:39:54.tar.gz

Reply:
I'll have another look next week and see if I can iron these things out.  It's really unfortunate that azure storage doesn't collapse those / characters and actually creates empty dir names.  But I can see some issues.

Some of your permutations (like bucket=master-data)/testfolder2) is never going to work because I think that isn't even a valid storage account name